invisible_cities/io/mcinfo_io.py

```diff
@@ -171,13 +175,73 @@ def __call__(self, mctables: (tb.Table, tb.Table, tb.Table, tb.Table),
         self.generator_table.flush()
 
 
-def copy_mc_info(h5in : tb.File, writer : Type[mc_info_writer], which_events : List[int]=None):
-    """Copy from the input file to the output file the MC info of certain events.
+def save_mc_table(file_in : str                         ,
```

Comment:
This seems rather complex, aren't we supposed to save all the output in a unique (new) MC format? ie, in copy_mc_info we read all relevant data (using our universal for both old and new readers) and just write the relevant tables? 
I would prefer to have output files with a unique structure even if some of the data was not previously saved in old MC format and we have to fill it with NaNs

Reply:
It does output a unique format that's the same for all but you have to take into account that the input isn't always the same and output it in a readable format